packages/ui-core/src/components/timezone/timezone.component.tsx

```diff
@@ -1,74 +1,136 @@
-import React, { FC, useMemo, useState } from 'react';
-import { Timezones } from '@keen.io/query';
+import React, { FC, useState, useRef, useEffect } from 'react';
+import { AnimatePresence } from 'framer-motion';
+import { FixedSizeList as ReactWindowList } from 'react-window';
 
 import Title from '../title';
-import DropdownList from '../dropdown-list';
-import DropdownListContainer from '../dropdown-list-container';
 import DropableContainer from '../dropable-container';
-import Dropdown from '../dropdown';
 
-import { Container, SelectContainer } from './timezone.styles';
+import {
+  Container,
+  SelectContainer,
+  ListItem,
+  Offset,
+  Name,
+  CustomDropdown,
+  DropableContent,
+} from './timezone.styles';
 
-import { TIMEZONES } from './constants';
+import { Timezone as TimezoneType, DropdownPosition, Options } from './types';
+import { dropdownMotion } from './motion';
 
 type Props = {
+  /** Timezones */
+  timezones: TimezoneType[];
   /** Timezone value */
-  timezone?: Timezones;
+  timezone?: TimezoneType;
```

Comment:
In my opinion we should not stick `1:1` with back-end schema. Current `timezone` value should be only `string` - that represents named timezone. Providing **utcOffset** just to set value should not be required a Explorer / Creator will be forced to persist it somewhere.